yt_dlp/extractor/acfun.py

```diff
@@ -185,7 +178,7 @@ def _real_extract(self, url):
         return {
             'id': video_id,
             'title': title,
-            'duration': int_or_none(video_info['durationMillis'], 1000),
+            'duration': int(video_info['durationMillis']) / 1000,
```

Comment:
No, i didn't mean like this, i mean you could write like this:
```python
'duration': int_or_none(video_info.get('durationMillis'), 1000)
```

Reply:
> No, i didn't mean like this, i mean you could write like this:

oh, sorry